src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -475,6 +476,22 @@ PARAM_DEFINE_FLOAT(FW_AIRSPD_MAX, 20.0f);
  */
 PARAM_DEFINE_FLOAT(FW_AIRSPD_TRIM, 15.0f);
 
+/**
+ * Stall Airspeed (CAS)
+ *
+ * The stall airspeed (calibrated airspeed) of the vehicle.
+ * It is used for airspeed sensor failure detection and for the control
+ * surface scaling airspeed limits.
+ *
+ * @unit m/s
+ * @min 0.5
+ * @max 40
+ * @decimal 1
+ * @increment 0.5
+ * @group FW TECS
+ */
+PARAM_DEFINE_FLOAT(FW_AIRSPD_STALL, 7.0f);
```

Comment:
previous stall speed param defaulted to 10 m/s, any reason you are lowering to 7 m/s?  In my experience 7 m/s is a very low stall speed, maybe many small hobby foamy airplanes (sometimes called "park fliers") with no payload could have stall speed this slow, but every fixed wing UAV I've worked with typically had a stall speed 9 m/s or greater (usually greater).  What is the safer default: a low value or a high value?

Reply:
I put it to 7m/s because FW_AIRSPD_MIN (being the min airspeed setpoint) is set to 10 by default, and it makes sense to have have some margin in between. 

>What is the safer default: a low value or a high value?

I would argue a lower value, because that reduces the amount of false positives of airspeed failure detected (given that this feature is anyway in still some kind of beta phase).